### PR TITLE
Add debug logging for Boss assignment

### PR DESCRIPTION
**Pull request by Toakan**
## What type of PR is this.
1. [X] Bug
2. [ ] Enhancement

### What have you changed and why?
Information:
    Added debug logging for the Boss generation, hopefully it can shed some light on why the boss fails to assign.

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the Mission in Singleplayer?
2. [X] Have you loaded the Mission in a Dedicated Server?

### Is further testing or are further changes required?
1. [ ] No
2. [X] Yes (Please provide further detail below.)

********************************************************
Notes:
This needs testing on the Official servers to determine what's going wrong with #512 .

### CODE REVIEW

File: A3-Antistasi/initPlayerLocal.sqf [hunk 1]
```diff
@@ -394,13 +408,12 @@ if (_isJip) then {
 	}
 	else {
 		hint format ["Welcome back %1", name player];
-		if ((isNil "theBoss" || {isNull theBoss}) && {{([_x] call A3A_fnc_isMember) and (side (group _x) == teamPlayer)} count playableUnits == 1}) then {
+		if (!(_BossAssigned) && {{([_x] call A3A_fnc_isMember) and (side (group _x) == teamPlayer)} count playableUnits == 1}) then {
```
Comment (Spoffy) [inline A3-Antistasi/initPlayerLocal.sqf]: Is this line redundant now?

Comment (Spoffy) [reply A3-Antistasi/initPlayerLocal.sqf]: (And the 2 below it)

Comment (Toakan) [reply A3-Antistasi/initPlayerLocal.sqf]: Ok slight problem...

This conflicts with the work i've done in #516

**Comment by Toakan** [comment]
Closing this as the work i've done in the later PR negates this work anyway.

File: A3-Antistasi/initPlayerLocal.sqf (hunk 1 above)
Comment (Toakan) [reply A3-Antistasi/initPlayerLocal.sqf]: Already resolved in #515

**Pull request merged**